### PR TITLE
Branch c

### PR DESCRIPTION
**Pull request by sky-blue2022**
**Key Changes**

1. Updated CSS on the landing page
2. Integrated API

### CODE REVIEW

File: git_remote_practice.html [hunk 1]
```diff
@@ -16,6 +16,8 @@ <h1>M3LA</h1>
 <!-- /Branch B-->
 
 <!-- Branch C-->
-N/A
+<h1>C1LB</h1>
+<h1>C2LB</h1>
```
Comment (bloovee) [inline git_remote_practice.html]: Found an error. Please fix this line.

File: git_remote_practice.html [hunk 2]
```diff
@@ -16,6 +16,8 @@ <h1>M3LA</h1>
...
+<h1>C3LB</h1>
```
Comment (bloovee) [inline git_remote_practice.html:21]: There is another error here. Please fix this line.

**Review by bloovee — CHANGES_REQUESTED** [review]
Overall looking good. Please fix the errors before merging the branch with the master.

**Comment by sky-blue2022** [comment]
Already fixed the errors and pushed new code. Please review and merge the code.

**Review by bloovee — APPROVED** [review]

**Pull request merged**